### DISCUSSION TITLE
Unable to access any url with non-ASCII characters in Codespaces in port forwarding

### DISCUSSION THREAD

**Discussion by Young-Lord** [community/community#59866] [Codespaces] [answered]
### Select Topic Area

Bug

### Body

Unable to access any url with non-ASCII characters in Codespaces in port forwarding

Reproduce:
1. Create a default codespace (for example: "literate spork" of @Young-Lord)
2. `mkdir test`
3. `cd test`
4. `mkdir "！！" "你好" "Hello" "こんにちは" "안녕하세요" "halló"`
5. `python -m http.server`
6. access port 8000, click on every folder hyperlink
7. HTTP 500 triggered. (or "Server error")

Related Request ID:
f2b31c55513aae38eacccc34c8de3527
3ed43a945d8af8ca01a9d40cfd5efde6
https://young-lord-literate-spork-xv6xv4g9jq734jp-8000.preview.app.github.dev/authenticate-codespace/young-lord-literate-spork-xv6xv4g9jq734jp?path=%2f%25EF%25BC%2581%25EF%25BC%2581%2f&visibility=private&port=8000&cid=2bc23d565b4faadc57158107a1e7411e ({"message":"Server error","stackTrace":null})

**Comment by Young-Lord** [comment]
similar feedback: <https://github.com/orgs/community/discussions/3748>, but seems inactive.

**Comment by samruddhikhandale** [comment, marked as answer]
Hi 👋 

Thanks for reporting the issue. We have a newer (experimental) version for public port forwarding which we plan to GA soon, more details in [here](https://github.com/orgs/community/discussions/57596#discussioncomment-6135903).

@Young-Lord can you remove the `preview` sub domain from your port forwarded URL and retry? It has the bug fix for this issue, thanks!

![image](https://github.com/community/community/assets/24955913/00bf9f3e-7459-44d8-be86-c98448fc74b5)

**Reply by Young-Lord** [reply]
I confirm using the forwarding domain app.github.dev without the preview fixes the issue. Thanks!
